cloudtools/start.py

```diff
@@ -70,7 +73,13 @@ def init_parser(parser):
     # initialization action flags
     parser.add_argument('--init', default='', help='Comma-separated list of init scripts to run.')
     parser.add_argument('--vep', action='store_true', help='Configure the cluster to run VEP.')
+    
+    # gcloud overrides
+    default_project = check_output(['gcloud', 'config', 'get-value', 'core/project']).rstrip()
```

Comment:
this is the only way I could find to get the defaults - they don't appear to be easily accessible by client libs, but I didn't want to break existing behavior where these aren't required